lib/index.js

```diff
@@ -29,10 +29,17 @@ export default function customElementToReact (elementClass, options = {}) {
   const skipProps = customProps.slice() // Keep a copy
   const tagName = `${dashCase}-${options.suffix || 'react'}`.replace(/\W+/g, '-').toLowerCase()
 
+  skipProps.push('customElementRef')
```

Comment:
`innerRef` is the more traditional name in the React community for this kind of forwarding ref prop.

Reply:
Changed the name to innerRef.